File: MT.Model/Business/ProjectMember.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectMember:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectMember:EntityBase.
    /// </summary>
    public class ProjectMember : EntityBase
    {
        [Key]
        public Guid ProjectMemberID { get; set; }
        public Guid ProjectID { get; set; }
        public Guid? EmployeeID { get; set; }
        public string FullName { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public decimal MonthForProject { get; set; }
        public Guid? ProjectPositionID { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        public string EmployeeName { get; set; }
        public string ProjectPositionName { get; set; }
        public int Rownum { get; set; }
        
    }
    #endregion
}

File: BECore/app/model/ProjectMember.js
/// <summary>
/// Model EXTJS ProjectMember.
/// </summary>
Ext.define('QLDT.model.ProjectMember', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectMemberID',
    idType: 'Guid',
    fields: [
		{ name: "ProjectID", type: "string" },
		{ name: "EmployeeID", type: "string" },
		{ name: "FullName", type: "string" },
		{ name: "StartDate", type: "date" },
		{ name: "EndDate", type: "date" },
		{ name: "MonthForProject", type: "float" },
		{ name: "ProjectPositionID", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "EmployeeName", type: "string" },
        { name: "ProjectPositionName", type: "string" },
        { name: "Rownum", type: "int" },
        
    ]
});

